translations/de/common.json

```diff
@@ -1 +1,81 @@
+  "information.rules": "Welche Regeln werden verwendet?",
+  "information.rules.message": "Im Moment verwendet die App die offiziellen EU-Eintrittsregeln, um das Zertifikat zu validieren. Diese Regeln werden von den Mitgliedsländern des digitalen COVID Zertifikats zur Verfügung gestellt. Es ist ein bekanntes Problem, dass diese Regeln veraltet sein können und nicht den staatlichen oder lokalen Vorschriften entsprechen. Dieses Projekt arbeitet an einem benutzerdefinierten Regelersteller, der präzisere Regeln für lokale Vorschriften ermöglicht. Sie können die Entwicklung<1>hier</1>verfolgen.",
+  "information.issue": "Was ist zu tun, wenn etwas nicht funktioniert?",
+  "information.issue.message": "Überprüfen Sie zunächst, ob Sie das richtige Land und die richtigen Regeln ausgewählt haben. Versuchen Sie dann, Ihr Gerät mit dem Internet zu verbinden, um die neusten Daten herunterzuladen. Wenn das Problem weiterhin auftritt, erstellen Sie auf<1>GitHub</1>ein Issue.",
+  "information.certificate": "Was ist das digitale COVID-Zertifikat der EU?",
+  "information.certificate.message": "Weitere Informationen dazu finden Sie auf<1>der offiziellen Website der EU</1>.",
+
+  "scan.button": "Zertifikat prüfen",
+  "scan.message": "Halten Sie den QR-Code vor Ihre Kamera",
+  "scan.error": "Kamera Fehler",
+  "scan.error.message": "Haben Sie Zugriff auf die Kamera erlaubt?",
+  "scan.multiscan": "Scannen Sie das Testzertifikat für",
+  "scan.note.permission": "Stellen Sie immer sicher, dass Sie berechtigt sind, das Zertifikat der anderen Person zu scannen!",
+  "scan.note.rules": "Überprüfen Sie die ausgewählten Regeln und stellen Sie sicher, dass Sie Ihren lokalen Vorschriften entsprechen. Andernfalls kann es zu ungültigen Prüfergebnissen kommen.",
+
+
+  "camera.close": "Kamera schließen",
+
+  "modal.purpose": "Prüfmodus auswählen",
+
+  "modal.rules": "Land und Regeln auswählen",
+  "modal.rules.updated": "aktualisiert",
+  "modal.rules.comingsoon": "Bald Verfügbar",
+  "modal.rules.confirmation": "Bitte stellen Sie sicher, dass diese Regeln Ihre Vorschriften entsprechen",
+  "modal.rules.confirmation.confirm": "Auswahl bestätigen",
+
+  "modal.settings": "Einstellungen",
+  "modal.settings.darkmode": "Dunkler Modus",
+  "modal.settings.language": "Bevorzugte Sprache",
+
+  "modal.result.valid": "GÜLTIG",
+  "modal.result.notvalid": "UNGÜLTIG",
+  "modal.result.testrequired": "TEST ZERTIFIKAT BENÖTIGT",
+  "modal.result.technicallyinvalid": "TECHNISCH UNGÜLTIG",
+  "modal.result.name": "Name",
+  "modal.result.birthdate": "Geburtsdatum",
+  "modal.result.rulecount": "Überprüfte Regeln",
+  "modal.result.scannext": "Zertifikat scannen",
+
+  "onboarding.message": "Covid Validator ermöglicht es Ihnen, <b>digitale COVID-Zertifikate der EU</b> zu scannen und zu validieren, basierend auf länderspezifischen oder lokalen Regeln. Der Scan- und Überprüfungsprozess findet lokal in Ihrem Browser statt.",
```

Comment:
Wie ist deine Meinung zu `... scannen und basierend auf länderspezifischen oder lokalen Regeln zu validieren.`

Reply:
Klingt besser!

Reply:
Geländert in [`0a59c8f`](https://github.com/timokoenig/covid-validator/pull/16/commits/0a59c8f0c5857c9b062b6da61d2ceea8d65d886f).